elro/device.py

```diff
@@ -48,8 +67,9 @@ def __init__(self, device_id, device_type):
         self._battery_level = -1
         self._signal_strength = -1
         self._device_state = ""
-        self.device_type = device_type
-        self.device_type_name = DeviceType(device_type).name
+        self.device_type = DeviceType(device_type_id)
+        self.device_type_id = device_type_id
```

Comment:
The name should indeed be taken from `self.device_type.name` to avoid data duplication and have things more manageable. 

For the id however it is not exactly duplicate data because one device type can be linked to multiple ids. So without the extra variable you can not differentiate between for example fire alarm 0005 and 100D. They would both report as 0005. It is true that we currently do not actually use this differentiation in the code but it is passed to the json output.